extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SimpleScheduler.java

```diff
@@ -154,7 +154,7 @@ public boolean isRunning() {
     }
 
     SimpleTrigger createTrigger(String invokerClass, CronParser parser, Scheduled scheduled, int nameSequence, Config config) {
```

Comment:
If we move the logic to the static method, we don't need to inject `Config` at all...

Reply:
Thanks for spotting! I've also removed the unused config from QuartzScheduler.